cmd/arcaflow/main.go

```diff
@@ -51,61 +53,50 @@ func main() {
 		Stdout:      os.Stderr,
 	})
 
+	defaultConfig := `
+log:
+  level: info
+deployers:
+  image:
+    deployer_name: podman
+    deployment:
+      imagePullPolicy: IfNotPresent
+logged_outputs:
+  error:
+    level: info`
+
```

Comment:
Did you consider adding the Python deployer in here? It would allow more situations to work with the default config. If that isn't reliable then we need to improve how it handles defaults.

Reply:
I didn't consider the python deployer, mainly because that has been mostly an edge use case for us and I don't think we've tested a lot of workflows with it. I do wonder whether the engine default deployer should be podman or docker (if not python).

Reply:
I thought about this and re-considered your question, and I've now added the python deployer alongside the image deployer, and I've also changed the default image deployer to docker to be more universally compatible and to match existing documentation better.

Reply:
Oof. Actually, this causes a potential problem because the engine fails any time the python deployer is defined in the config if the python binary is not found.

Reply:
Added issue #190